test/unit/specs/components/common/SessionAccounts.spec.js

```diff
@@ -19,7 +19,7 @@ describe(`SessionAccounts`, () => {
         ]
       }
     }
-    wrapper = shallowMount(SessionAccounts, {
+    wrapper = mount(SessionAccounts, {
```

Comment:
Why `mount` > `shallowMount`?

Reply:
I'm guessing because the `AccountList` isn't rendered, which is what is tested. Maybe it's better to ShallowMount and test that the accounts are present and passed to the `AccountList`?

@jbibla I can't comment on the line because it isn't diffed, but does the computed property `accounts` do anything? It seems to pass the same array?

Reply:
I'm guessing because the `AccountList` isn't rendered, which is what is tested. Maybe it's better to ShallowMount and test that the accounts are present and passed to the `AccountList`?